src/cp.ts

```diff
@@ -65,7 +64,7 @@ export class Cp {
      * @param {string} containerName - The name of the container in the pod to exec the command inside.
      * @param {string} srcPath - The source path in local
      * @param {string} tgtPath - The target path in the pod
-     * @param {string} cwd - The directory that is used as the parent in the host when uploading
+     * @param {string} [cwd] - The directory that is used as the parent in the host when uploading
```

Comment:
Why did you change the `cwd` to be in square brackets?
Doesn't that mean it's an array of strings while it isn't?

Reply:
When a param name is in the square brackets, it means that it is optional

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

Reply:
Thank you I did not know this.